geomesa-quickstart-kafka/geomesa-quickstart-kafka-08/src/main/java/com/example/geomesa/kafka08/KafkaLoadTester.java

```diff
@@ -177,8 +185,24 @@ public static void main(String[] args) throws Exception {
                     }
             );
 
-            System.out.println("Consumer has size: " + consumerFS.getFeatures().size());
-            System.out.println("At " + new Date() + " sleeping before writing more features.");
+            // count features written
+            Integer consumerSize = consumerFS.getFeatures().size();
+            cycle++;
+            featuresSinceStartTime += consumerSize;
```

Comment:
Does the consumer fs drop features ever? Otherwise, won't it keep accumulating features, causing you to double count?